operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ Entries should include a reference to the Pull Request that introduced the chang
 ## main / unreleased
 
 * [CHANGE] Memcached: Update to Memcached 1.6.34. #10318
+* [ENHANCEMENT] Add support for individual components to override their container image. #10340
```

Comment:
i've been trying to make the changelog enough for anyone to get the gist of a change and know how to use it. Can you give an example of overriding the image for a component? Something like `ingester.image.tag` and `ingester.image.repository`. Or explain that there's an `image` object under all component objects now

Reply:
I've rephrased the entry with more details about the change.